universal-login-sdk/test/core/services/WalletService.int.test.ts

```diff
@@ -83,7 +83,7 @@ describe('INT: WalletService', async () => {
       await walletService.initializeConnection(ensName);
       walletService.waitForConnection();
```

Comment:
And then try to remove from WalletService
```ts
await waitUntil(() => !!this.getConnectingWallet().unsubscribe, tick, timeout);
```

```suggestion
      await walletService.waitForConnection();
```